packages/babel-helper-function-name/src/index.js

```diff
@@ -45,6 +45,18 @@ const visitor = {
   },
 };
 
+function getNameFromLiteralId(id) {
+  if (t.isNullLiteral(id)) {
+    return null;
+  } else if (t.isRegExpLiteral(id)) {
+    return `_${id.pattern}_${id.flags}`;
+  } else if (t.isTemplateLiteral(id) && id.expressions.length === 0) {
+    return id.quasis[0].value.raw;
+  } else {
+    return id.value;
```

Comment:
Given the name of this function, I think we should make sure that this is converted to a string before returning it.